scripts/baseline_comparison/evaluate_utils.py

```diff
@@ -183,17 +183,26 @@ def generate_sensitivity_plots(df, show: bool = False, save_prefix: str = None):
                 df_portfolio_agg = df_portfolio.loc[df_portfolio["is_ensemble"] == is_ens].copy()
                 df_portfolio_agg = df_portfolio_agg[[dimension, metric, "seed"]].groupby([dimension, "seed"]).mean()[metric]
                 dim, mean, sem = df_portfolio_agg.groupby(dimension).agg(["mean", "sem"]).reset_index().values.T
+                _, _, std = df_portfolio_agg.groupby(dimension).agg(["mean", "std"]).reset_index().values.T
```

Comment:
Why not reporting the standard error? If we want to report confidence intervals, we should use sem.

Reply:
Good point, I've updated back to sem